src/windows/WATAppBar.js

```diff
@@ -28,7 +28,8 @@ addAppBar = function () {
   WinJS.UI.processAll().then(function () {
     var div = document.createElement("div");
     div.style.zIndex = WAT.components.webView.style.zIndex + 100;
-
+    div.classList.add("customColor");
```

Comment:
I would use a more specific class name like: appbarCustomization or something like that. However, we should think if we would want to add a class or just apply the style to the element as we already have the reference in WAT.components.appBar. The only problem with that approach is that the appBar should be initialized before the styles. Thoughts?